### PR TITLE
Fix `ASTAlterCommand::formatImpl` in case of column specific settings…

### PR DESCRIPTION
**Pull request by antaljanosbenjamin**
### Changelog category (leave one):
- Bug Fix (user-visible misbehavior in an official stable release)

### Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
- Fix formatting of alter commands in case of column specific settings 

### Documentation entry for user-facing changes

- [ ] Documentation is written (mandatory for new features)

### CODE REVIEW

**Comment by robot-clickhouse-ci-1** [comment]
*This is an automated comment for commit 94e4c42cab4f89a27f9bb7644857b5009bd0b781 with description of existing statuses. It's updated for the latest CI running*

[⏳ Click here](https://s3.amazonaws.com/clickhouse-test-reports/59445/94e4c42cab4f89a27f9bb7644857b5009bd0b781/ci_running.html) to open a full report in a separate page

<table>
<thead><tr><th>Check name</th><th>Description</th><th>Status</th></tr></thead>
<tbody>
<tr><td>CI running</td><td>A meta-check that indicates the running CI. Normally, it's in <b>success</b> or <b>pending</b> state. The failed status indicates some problems with the PR</td><td><a href="https://s3.amazonaws.com/clickhouse-test-reports/59445/94e4c42cab4f89a27f9bb7644857b5009bd0b781/ci_running.html">⏳ pending</a></td></tr>
<tbody>
</table>

File: src/Parsers/ASTAlterQuery.cpp [hunk 1]
```diff
@@ -104,6 +104,14 @@ void ASTAlterCommand::formatImpl(const FormatSettings & settings, FormatState &
...
             settings.ostr << (settings.hilite ? hilite_keyword : "") << " REMOVE " << remove_property;
         }
+        else if (settings_changes) {
+            settings.ostr << (settings.hilite ? hilite_keyword : "") << "MODIFY SETTING " << (settings.hilite ? hilite_none : "");
+            settings_changes->formatImpl(settings, state, frame);
+        }
+        else if (settings_resets) {
+            settings.ostr << (settings.hilite ? hilite_keyword : "") << "RESET SETTING " << (settings.hilite ? hilite_none : "");
+            settings_resets->formatImpl(settings, state, frame);
+        }
```
Comment (evillique) [inline src/Parsers/ASTAlterQuery.cpp:114]: Style + spaces in keywords
```suggestion
        else if (settings_changes)
        {
            settings.ostr << (settings.hilite ? hilite_keyword : "") << " MODIFY SETTING " << (settings.hilite ? hilite_none : "");
            settings_changes->formatImpl(settings, state, frame);
        }
        else if (settings_resets)
        {
            settings.ostr << (settings.hilite ? hilite_keyword : "") << " RESET SETTING " << (settings.hilite ? hilite_none : "");
            settings_resets->formatImpl(settings, state, frame);
        }
```

**Comment by antaljanosbenjamin** [comment]
@canhld94 As you are the original author, feel free to have a look at this, maybe you can spot something that I missed.

**Review by evillique — APPROVED** [review]

**Review by rschu1ze — APPROVED** [review]

**Comment by canhld94** [comment]
@antaljanosbenjamin thank you for fixing this. I don't have further comment.

**Comment by antaljanosbenjamin** [comment]
The integration test is flaky ([see](https://play.clickhouse.com/play?user=play#U0VMRUNUIGNoZWNrX3N0YXJ0X3RpbWUsIGNoZWNrX25hbWUsIHRlc3RfbmFtZSwgdGVzdF9zdGF0dXMsIGNoZWNrX3N0YXR1cywgcmVwb3J0X3VybApGUk9NIGNoZWNrcwpXSEVSRSAxCiAgICBBTkQgcHVsbF9yZXF1ZXN0X251bWJlciA9IDAKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnU0tJUFBFRCcKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnT0snCiAgICBBTkQgY2hlY2tfc3RhdHVzICE9ICdzdWNjZXNzJwogICAgQU5EIHRlc3RfbmFtZSBpbGlrZSAnJXRlc3Rfa2VlcGVyX2Rpc2tzL3Rlc3QucHk6OnRlc3RfbG9nc193aXRoX2Rpc2tzJScKT1JERVIgQlkgY2hlY2tfc3RhcnRfdGltZSBkZXNjLCBjaGVja19uYW1lLCB0ZXN0X25hbWU=)), while the integration test failed because the cleanup of replicated database tests are flaky in general ([see](https://play.clickhouse.com/play?user=play#U0VMRUNUIGNoZWNrX3N0YXJ0X3RpbWUsIGNoZWNrX25hbWUsIHRlc3RfbmFtZSwgdGVzdF9zdGF0dXMsIGNoZWNrX3N0YXR1cywgcmVwb3J0X3VybCwgdGVzdF9jb250ZXh0X3JhdwpGUk9NIGNoZWNrcwpXSEVSRSAxCiAgICAtLSBBTkQgcHVsbF9yZXF1ZXN0X251bWJlciA9IDAKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnU0tJUFBFRCcKICAgIEFORCB0ZXN0X3N0YXR1cyAhPSAnT0snCiAgICBBTkQgY2hlY2tfc3RhdHVzICE9ICdzdWNjZXNzJwogICAgQU5EIHRlc3RfY29udGV4dF9yYXcgaWxpa2UgJyVDYW5ub3QgZW5xdWV1ZSBxdWVyeSwgYmVjYXVzZSBzb21lIHJlcGxpY2EgYXJlIHRyeWluZyB0byBlbnF1ZXVlIGFub3RoZXIgcXVlcnkuIEl0IG1heSBoYXBwZW4gb24gaGlnaCBxdWVyaWVzIHJhdGUgb3IsIGluIHJhcmUgY2FzZXMsIGFmdGVyIGNvbm5lY3Rpb24gbG9zcy4lJwpPUkRFUiBCWSBjaGVja19zdGFydF90aW1lIGRlc2MsIGNoZWNrX25hbWUsIHRlc3RfbmFtZQ==)).

**Pull request merged**